agent/php_user_instrument.c

```diff
@@ -47,6 +47,9 @@
  *
  * Many functions here call zend_bailout to continue handling fatal PHP errors,
  * Since zend_bailout calls longjmp it never returns.
+ *
+ * Note: The agent ONLY needs to call this if it has overwritten the original
+ * zend_execute_ex.
```

Comment:
Can we add a guard to this function to have it exit harmlessly if OAPI is enabled - or is this even a good idea?

Reply:
It will exit harmlessly since orig_execute points to a no_op.  This way we maintain backward compatibility when frameworks call it from within instrumentation.